recipes-bsp/imx-atf/imx-atf_1.4.1.bb

```diff
@@ -0,0 +1,65 @@
+# Copyright 2017-2018 NXP
+
+DESCRIPTION = "i.MX ARM Trusted Firmware"
+SECTION = "BSP"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/files/common-licenses/BSD-3-Clause;md5=550794465ba0ec5312d6919e203a55f9"
+
+PV = "1.4.1+git${SRCPV}"
+
+SRCBRANCH = "imx_4.9.88_imx8qxp_beta2"
```

Comment:
Why not using the 4.9.123 2.3.0 release tag (as the series title suggests)?
That would also mean adding SOC_ATF_mx8mm below.